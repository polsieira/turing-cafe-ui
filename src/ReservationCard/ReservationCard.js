import React from 'react';
import '../ReservationCard/ReservationCard';

const ReservationCard = ({ name, date, time, numberOfGuests }) => {

  return (
    <>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time}</p>
      <p>{`Number of guests: ${numberOfGuests}`}</p>
      <button type='button'>Cancel</button>
    </>
  )

}

export default ReservationCard;
