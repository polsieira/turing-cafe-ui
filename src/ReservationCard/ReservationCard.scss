.card {
  background: white;
  border: teal 5px solid;
  border-radius: 15px;
  color: teal; 
  margin: 10px;
  padding: 15px;
}

.cancel-btn {
  background: teal;
  color: white;
  font-size: 20px;
  height: 30px;
  width: 90%;
}