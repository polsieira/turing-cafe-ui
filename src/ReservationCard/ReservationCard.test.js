import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import ReservationCard from './ReservationCard';

describe('ReservationCard', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow()
  })
});