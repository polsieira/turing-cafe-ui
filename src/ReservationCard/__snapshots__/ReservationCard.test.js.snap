// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationCard should match a snapshot 1`] = `
<div
  className="card"
>
  <h2>
    Pol
  </h2>
  <p>
    10/17
  </p>
  <p>
    7:00 pm
  </p>
  <p>
    Number of guests: 10
  </p>
  <button
    className="cancel-btn"
    id={1}
    onClick={[Function]}
    type="button"
  >
    Cancel
  </button>
</div>
`;
